pandas/compat/__init__.py

```diff
@@ -65,3 +66,27 @@ def is_platform_mac():
 
 def is_platform_32bit():
     return struct.calcsize("P") * 8 < 64
+
+
+def _import_lzma():
+    try:
+        import lzma
+
+        return lzma
+    except ImportError:
+        msg = (
+            "Could not import the lzma module. "
+            "Your installed Python is incomplete. "
+            "Attempting to use lzma compression will result in a RuntimeError."
+        )
+        warnings.warn(msg)
+
+
+def _get_lzma_file(lzma):
+    if lzma is None:
+        raise RuntimeError(
```

Comment:
this seems overly complicated; why don't we just do this with one function? e.g. import in the get_lzma_file and warn there

Reply:
I think we want a warning an import time right? To let people know that their Python install is incomplete?

Reply:
I c, I guess this is ok then.